config/resources/namespace/namespace.yaml

```diff
@@ -3,4 +3,4 @@ kind: Namespace
 metadata:
   labels:
     control-plane: controller-manager
-  name: system
+  name: cluster-api-provider-packet-system
```

Comment:
I believe this should likely be replaced using kustomize rather than hardcoding it here. I'll try to compare against the other providers to see if I can figure out what is different here.